### PR TITLE
Wasm bindgen example

### PR DESCRIPTION
**Pull request by wa1aric**
#394

### CODE REVIEW

File: examples/js-snippets/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        span {(*product.get())}
+    }
+}
+
+fn main() {
+    sycamore::render(|cx| {
+        view! {
+            cx,
+            App
+        }
```
Comment (lukechu10) [inline examples/js-snippets/src/main.rs:34]: Nit:
```suggestion
        view! { cx,
            App
        }
```

File: examples/js-snippets/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    });
+
+    view! {
+        cx,
+        input(type="number", bind:valueAsNumber=a)
+        span {"*"}
+        input(type="number", bind:valueAsNumber=b)
+        span {"="}
+        span {(*product.get())}
+    }
```
Comment (lukechu10) [inline examples/js-snippets/src/main.rs:26]: Nit: formatting
```suggestion
    view! { cx,
        input(type="number", bind:valueAsNumber=a)
        span { "*" }
        input(type="number", bind:valueAsNumber=b)
        span { "=" }
        span { (*product.get()) }
    }
```

**Review by lukechu10 — COMMENTED** [review]
Thanks a lot!
Just two small formatting problems with the view! macro to make the style consistent with the rest of the codebase.

Also it might be better to name the JS file `index.js` instead of `functions.js` but that is a really small detail and probably doesn't really matter.

**Comment by lukechu10** [comment]
Also just realized this. Could you please add an entry to `examples/README.md` in alphabetical order? Thanks!

**Comment by lukechu10** [comment]
It seems like you haven't fully fixed the formatting. Notice the space after the `{` and before the `}`. Apart from that, everything looks good.

**Comment by wa1aric** [comment]
Done!.

**Pull request merged**